user-info-api/pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>fwu-kc-extensions</artifactId>
+    <groupId>de.intension</groupId>
+    <version>${projectVersion}</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>user-info-api</artifactId>
```

Comment:
Maybe rename all folders/names to `person-info`? Better now than later

Reply:
Discussed with the customer -> we will keep it as it is